### PR TITLE
Search box new style

### PR DESCRIPTION
**Pull request by aosyatnik**
### Purpose

Link to YouTrack:
[MAGN-8385](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8385) Update Library Search box UI so that it stands out

Results:
image

On hover
image

Text
image

### Declarations

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files

### Reviewers

?

### FYIs

@Racel

### CODE REVIEW

**Comment by Racel** [comment]
Thanks, @aosyatnik. Can you make the text box a little bit taller and ensure that the text and icon are horizontally centered in the box? Looks like the text is slightly lower than the search icon.

**Comment by aosyatnik** [comment]
@Racel , what about now? I've made search box 2 pixels taller.
image

I didn't change horizontal/vertical alignment. Maybe it's because "poin" contains only low letters... With "list" everything seems ok.

**Comment by Racel** [comment]
Thanks @aosyatnik - looks much better now! LGTM.

**Comment by mjkkirschner** [comment]
+1

**Comment by jnealb** [comment]
1+

File: src/DynamoCoreWpf/UI/Themes/Modern/DynamoColorsAndBrushes.xaml [hunk 1]
```diff
@@ -80,7 +80,8 @@
 
     <!-- Library text colors-->
     <SolidColorBrush x:Key="MemberButtonText" Color="#aaaaaa" />
-    <SolidColorBrush x:Key="DeafultSearchTextBlockText" Color="#878787" />
+    <SolidColorBrush x:Key="DeafultSearchTextBlockText"
+                     Color="#6D6E71" />
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/UI/Themes/Modern/DynamoColorsAndBrushes.xaml]: can you replace the constant colors with named versions like exited previously? maybe we'll reuse these somewhere.

File: src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml [hunk 2]
```diff
@@ -124,14 +126,14 @@
...
-                               Margin="3,0,0,0">
+                               Margin="6,0,0,0">
 
                     </TextBlock>
                 </StackPanel>
 
                 <TextBox Name="SearchTextBox"
                          KeyboardNavigation.TabIndex="0"
-                         Foreground="{StaticResource CommonSidebarTextColor}"
+                         Foreground="#231F20"
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml]: can you replace the string with named versions like existed previously? maybe we'll reuse these somewhere.

**Comment by mjkkirschner** [comment]
@aosyatnik this looks good to merge except for one tiny thing, I think it would be good to keep names for some of the colors if we reuse them. It'll also help document the xaml.

**Comment by mjkkirschner** [comment]
@aosyatnik thanks, LGTM

**Pull request merged**